### PR TITLE
Add privacy and CCPA policies

### PR DESCRIPTION
**Pull request by cairomassimo**

### CODE REVIEW

**Review by Shadowfiend — COMMENTED** [review]
Quick thing to fill in the rest of a year, rest of this looks fine pending the build passing.

File: src/pages/ccpa.mdx [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import LegalLayout from "../layout/legal-layout.tsx";
+export default LegalLayout;
+
+## Privacy Notice for California Residents
+
+## Effective Date:\*\* February 21, 20
```
Comment (Shadowfiend) [inline src/pages/ccpa.mdx:6]:
```suggestion
## Effective Date:\*\* February 21, 2022
```

Comment (cairomassimo) [reply src/pages/ccpa.mdx:6]: Thanks!

**Review by Shadowfiend — APPROVED** [review]
Looks good, let's do it.

**Pull request merged**